src/sentry/integrations/slack/integration.py

```diff
@@ -62,24 +62,43 @@ class SlackIntegrationProvider(IntegrationProvider):
     features = frozenset([IntegrationFeatures.CHAT_UNFURL, IntegrationFeatures.ALERT_RULE])
 
     # Scopes differ depending on if it's a workspace app
-    identity_oauth_scopes = (
-        frozenset(["bot", "links:read", "links:write"])
-        if not settings.SLACK_INTEGRATION_USE_WST
-        else frozenset(
-            [
-                "channels:read",
-                "groups:read",
-                "users:read",
-                "chat:write",
-                "links:read",
-                "links:write",
-                "team:read",
-            ]
-        )
+    wst_oauth_scopes = frozenset(
+        [
+            "channels:read",
+            "groups:read",
+            "users:read",
+            "chat:write",
+            "links:read",
+            "links:write",
+            "team:read",
+        ]
+    )
+
+    # some info here: https://api.slack.com/authentication/quickstart
+    bot_oauth_scopes = frozenset(
+        [
+            "channels:read",
+            "groups:read",
+            "users:read",
+            "chat:write",
+            "links:read",
+            "links:write",
+            "team:read",
+            "chat:write.public",
+            "chat:write.customize",
+        ]
```

Comment:
Is this going to break things for on prem users using the bot token apps?

Reply:
@EvanPurkhiser No, this will not. Existing bot installs that are classic bots have the `bot` scope which is very expansive and includes all these scopes.

Reply:
ah ok I see